pages/awards/index.js

```diff
@@ -0,0 +1,335 @@
+import FilterDropdown from '@components/FilterDropdown'; // Filter Dropdown
+import Table from '@components/Table'; // Table
+import Pill from '@components/Pill'; // Pill
+import Pagination from '@components/Pagination'; // Pagination
+import BackArrow from '@assets/back-arrow.svg'; // Back arrow icon
+import Search from '@assets/search.svg'; // Search icon
+import ChevronDown from '@assets/chevron-down.svg'; // Chevron down icon
+import ChevronDownGrey from '@assets/chevron-down-grey.svg'; // Chevron down grey icon
+import DancerRedJump from '@assets/dancer-red-jump.svg'; // Jumping Dancer SVG
+import DancerYellowBlue from '@assets/dancer-yellow-blue.svg'; // Jumping Dancer SVG
+import styles from '@styles/pages/Performances.module.scss'; // Page styles
+
+// Temp constants
+import data, { columns } from '../../data/mockAwards';
+
+const PAGE_SIZE = 6; // Rows per page
+
+// Get the active filters (list of column accessors) from an object of filter dropdown values
+const getActiveFilters = options => {
+  return Object.keys(options).filter(option => options[option].selected);
+};
+
+// Remove key from object (returns new object)
+const removeKeyFromObject = (object, key) => {
+  // eslint-disable-next-line no-unused-vars
+  const { [key]: _, ...rest } = object;
+  return rest;
+};
+
+// Page: Performances
+export default function Performances() {
+  const [modalOpen, setModalOpen] = useState(false);
+  const [showFilters, setShowFilters] = useState(false);
+  const [query, setQuery] = useState('');
+  // Filter Options
+  const [awardTypeOptions, setAwardTypeOptions] = useState({});
+  const [danceTypeOptions, setDanceTypeOptions] = useState({});
+  // All Filter
+  const [filters, setFilters] = useState([]);
+  // Pagination
+  const [pageNumber, setPageNumber] = useState(0);
+  const [pageCount, setPageCount] = useState(0);
+
+  // TODO: Get filter options from Awards API
+  /*
+  // Get initial filter options
+  useEffect(() => {
+    const getSettings = async () => {
+      // Get settings values
+      const settingsResponse = await axios({
+        method: 'GET',
+        url: '/api/settings/collect',
+      });
+      const settings = settingsResponse.data;
+      const awardTypeSettings = {};
+      const danceTypeSettings = {};
+      for (const setting of settings) {
+        switch (setting.type) {
+          case 'COMPETITION_LEVEL': // TODO: AWARD TYPES
+            performanceLevelSettings[setting.value] = {
+              label: setting.value,
+              selected: false,
+            };
+            break;
+          case 'STYLE': // TODO: DANCE TYPE
+            danceStyleSettings[setting.value] = {
+              label: setting.value,
+              selected: false,
+            };
+            break;
+          default:
+            throw new Error('Invalid setting type');
+        }
+      }
+
+      setAcademicLevelOptions(academicLevelSettings);
+    };
+
+    getSettings();
+  }, []);
```

Comment:
We can just remove this commented code. I refactored it in https://github.com/uwblueprint/dancefest-web/pull/212.